R/ggplotly.R

```diff
@@ -217,8 +217,24 @@ gg2list <- function(p){
     }
     
     # This extracts essential info for this geom/layer.
-    traces <- layer2traces(L, df, misc)
-
+    if (L$geom$objname == "smooth") {
```

Comment:
The code in trace_generation.R seems to assume a layer consists of one geom (is that right?), but a `geom_smooth()` is really `geom_path() + geom_ribbon()`. I suppose I could add `smooth` (instead of `smoothRibbon` & `smoothLine`) to these lists, but the downside is that we essentially have to copy/paste over `ribbon` and `line` into this spot (so, if we happen to fix/change `ribbon` or `line`, those changes won't propagate to `smooth`). Would you prefer to take this approach instead? Or maybe there is a better way?